index.html

```diff
@@ -67,10 +69,15 @@ <h2>State of the Map Asia will bring together participants from around the world
     <!-- SECTION: ORGANISERS -->
     <div class="">
         <div class="w-75 px-3 pt-3 mx-auto text-center">
-            <h1 class="display-4">Organised By</h1>
+            <h1 class="display-5">Organised by</h1>
         </div>
         <div class="container">
             <div class="d-md-flex flex-md-equal w-100 my-md-3 pl-md-3">
+                <div class="mr-md-3 pt-3 px-3 px-md-5 text-center overflow-hidden">
```

Comment:
@geohacker Can you change `px-md-5` to `px-md-3` and see. I think that looks better for me.

Reply:
don't think that worked @tridip1931